udata_front/theme/gouvfr/datagouv-components/src/components/ReuseCard/ReuseCard.vue

```diff
@@ -109,3 +108,9 @@ const { label: reuseType } = useReuseType(() => props.reuse.type);
   object-fit: cover;
 }
 </style>
+<style scoped>
+/* From https://stackoverflow.com/a/31719534 */
+.wrapper-ellipsis-flow {
```

Comment:
Do we want to move this outside of both scoped style ?

Reply:
Don't know, what do you prefer? It's quite specific, maybe wait for the third repetition?